resources/views/registerProject.blade.php

```diff
@@ -197,10 +197,10 @@ function updateImage(imageInput) {
 				]
 			},
 			category:{
-				identifier:'category',
+				identifier:'category[]',
 				rules:[{
-						type:'empty',
-						prompt:'Please select a category'
+						type:'minCount[1]',
```

Comment:
non-blocking thought: hmmm I'm not sure this is true? ahora que son tags, me imagino que pueden estar vacías? 💭 lo mismo en el back-end validation.

Reply:
Pues en mi opinion muchos fields podrian ser opcionales  cuando agreguemos una vista de "update project", pero como no la tenemos pues creo que esta bien forzar a que el usuario llene todo (for now).